zotero.org.js

```diff
@@ -35,21 +35,20 @@
   ***** END LICENSE BLOCK *****
 */
 
-
-var sessionKey;
+/* eslint consistent-return: 0 */
```

Comment:
Better to leave this on and just return `false` if the function otherwise returns a value (e.g., no results from `detectWeb()`), so it still catches incorrect branching. (Despite the previous code, `selectItems()` doesn't actually need a return value, so that can just be `return;`.)